yorkie/src/main/kotlin/dev/yorkie/core/Client.kt

```diff
@@ -180,9 +182,8 @@ public class Client @VisibleForTesting internal constructor(
     public fun syncAsync(): Deferred<Boolean> {
         return scope.async {
             var isAllSuccess = true
-            attachments.value.map { (_, attachment) ->
-                attachment.document
-            }.asSyncFlow().collect { (document, result) ->
+            val attachments: List<Attachment> = attachments.value.values.toList()
```

Comment:
we don't need to make values to a list.
we can just change `asSyncFlow`'s receiver's type to `Collection<Attachment> `:)